homework-2/config/constants.ts

```diff
@@ -0,0 +1,45 @@
+import { User } from "../interfaces/User";
+
+export const APP_PORT = 3000;
+export const RES_STATUS_CODES = {
```

Comment:
I would use [http-status](https://www.npmjs.com/package/http-status) instead of custom constants.

Reply:
Done, thanks